website/docs/d/identitystore_groups.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "SSO Identity Store"
+layout: "aws"
+page_title: "AWS: aws_identitystore_groups"
+description: |-
+  Terraform data source for managing an AWS SSO Identity Store Groups.
+---
+
+# Data Source: aws_identitystore_groups
+
+Terraform data source for managing an AWS SSO Identity Store Groups.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+data "aws_ssoadmin_instances" "example" {}
+
+data "aws_identitystore_groups" "example" {
+  identity_store_id = tolist(data.aws_ssoadmin_instances.example.identity_store_ids)[0]
```

Comment:
Was there a particular reason this is converted to a list? I'm curious if this works:

```suggestion
  identity_store_id = data.aws_ssoadmin_instances.example.identity_store_ids[0]
```

Reply:
This is incorporated in the tests and probably based on the documentation of the `aws_ssoadmin_instances` docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ssoadmin_instances.

Would not have done it myself this way as well, inherited from the original PR. I'll check whether we can remove the `tolist` function

Reply:
If removing the `tolist` works, I'll create a follow-up PR to address that example in the docs 👍🏼

Reply:
Validated, removed ✅

Reply:
Awesome